src/index.es6

```diff
@@ -20,7 +20,7 @@
 
 import React from 'react';
 import Application from './app.jsx';
-// import Overview from './app.jsx';
+
```

Comment:
This line is not in master, so some other commit of your's in this PR already adds this. These "fixups" should be squashed into the commit that they fix. The goal is a PR is a series of self-contained, correct, and well-described commits, which make sense by themselves, and can be reviewed and understood.